src/build.py

```diff
@@ -343,6 +344,8 @@ def GitCloneFetchCheckout(self):
     """Clone a git repo if not already cloned, then fetch and checkout."""
     if os.path.isdir(self.src_dir):
       print '%s directory already exists' % self.name
+      proc.check_call(['git', 'remote', 'set-url', 'origin', self.git_repo],
```

Comment:
Right, the intent was that if you're not doing a clean fresh-from-upstream build, that if we changed the origin in the script, this would update it for you. On reflection that might not be desired if the user had overridden origin intentionally.

Possibly we could set up a new remote, say 'waterfall', that the script uses and overwrites, and local users are free to manually sync an unchanging origin.